src/display.rs

```diff
@@ -178,8 +180,7 @@ impl Display {
             height: viewport_size.height.0 as f32,
             cell_width: cell_width as f32,
             cell_height: cell_height as f32,
-            padding_x: config.padding().x as f32,
-            padding_y: config.padding().y as f32,
+            padding: padding.clone(),
```

Comment:
Cloning the padding seems like a waste. It should be perfectly fine for padding to implement `Copy`.